althea_kernel_interface/src/hardware_info.rs

```diff
@@ -345,6 +349,70 @@ fn get_wifi_devices() -> Vec<WifiDevice> {
     ret
 }
 
+fn get_conntrack_info() -> Option<ConntrackInfo> {
+    const MAX_PATH: &str = "/proc/sys/net/netfilter/nf_conntrack_max";
+    const TABLE_COUNT_PATH: &str = "/proc/sys/net/netfilter/nf_conntrack_count";
```

Comment:
can you put a doc comment here about what each value represents?

Reply:
Fixed